test-suite/example-contracts/counter/script/grainfind.js

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env node
 
-const path = require('path');
-const cp = require('child_process');
-const which = require('which');
+const path = require("path");
```

Comment:
Can we drop the grainfind file now that the LSP does everything it used to do?

Reply:
Sure but I think that's out of scope of this PR